### PR TITLE
Implement bursaries.yml as variables

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Trello card

https://trello.com/c/2WHavuTv

### Context

- We're centralising all of our monetary values into one place. This PR does it for our bursaries.

### Changes proposed in this pull request

- Implement centralised variables for bursary related values

### Guidance to review

Pages where changes have been made:

[ ] https://get-into-teaching-app-review-4029.test.teacherservices.cloud/funding-and-support/scholarships-and-bursaries
[ ] https://get-into-teaching-app-review-4029.test.teacherservices.cloud/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=english&button=#funding-widget (Funding widget - test with each subject)
[ ] https://get-into-teaching-app-review-4029.test.teacherservices.cloud/is-teaching-right-for-me/computing
[ ] https://get-into-teaching-app-review-4029.test.teacherservices.cloud/is-teaching-right-for-me/engineers-teach-physics
[ ] https://get-into-teaching-app-review-4029.test.teacherservices.cloud/is-teaching-right-for-me/maths
[ ] https://get-into-teaching-app-review-4029.test.teacherservices.cloud/is-teaching-right-for-me/physics
[ ] https://get-into-teaching-app-review-4029.test.teacherservices.cloud/landing/how-to-fund-your-teacher-training
[ ] https://get-into-teaching-app-review-4029.test.teacherservices.cloud/non-uk-teachers/fees-and-funding-for-non-uk-trainees

### Pre-election period restrictions

* [ ] This change is permitted within the constraints of the [pre-election period guidelines](https://www.gov.uk/government/publications/election-guidance-for-civil-servants/general-election-guidance-2024-guidance-for-civil-servants-html#publishing-content-online-)

### CODE REVIEW

**Comment by Sarah-DfE** [comment]
@gemmadallmandfe all looks good to me on this one

**Review by Sarah-DfE — DISMISSED** [review]
all good

File: app/views/content/funding-and-support/scholarships-and-bursaries.md [hunk 1]
```diff
@@ -43,19 +43,19 @@ You cannot receive a scholarship or bursary if you're on a [salaried teacher tra
...
-| **Geography**                                                         | £25,000  |                  |
-| **Languages**<br/> (French, German and Spanish only)                  | £25,000  | $scholarships_languagesfrenchgermanspanish$      |
-| **Languages**<br/> (all other languages, including ancient languages) | £25,000  |                  |
-| **Maths**                                                             | £28,000  | $scholarships_maths$         |
-| **Music**                                                             | £10,000  |                  |
-| **Physics**                                                           | £28,000  | $scholarships_physics$        |
-| **Religious education**                                               | £10,000  |                  |
+| **Art and design**                                                    | $bursaries_postgraduate_artanddesign$  |                  |
+| **Biology**                                                           | $bursaries_postgraduate_biology$  |                  |
+| **Chemistry**                                                         | $bursaries_postgraduate_chemistry$  | £30,000          |
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/scholarships-and-bursaries.md:48]:
```suggestion
| **Chemistry**                                                         | $bursaries_postgraduate_chemistry$  | $scholarships_chemistry$          |
```

File: app/views/content/funding-and-support/scholarships-and-bursaries.md [hunk 2]
```diff
@@ -43,19 +43,19 @@ You cannot receive a scholarship or bursary if you're on a [salaried teacher tra
...
+| **Computing**                                                         | $bursaries_postgraduate_computing$  | £30,000          |
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/scholarships-and-bursaries.md:49]:
```suggestion
| **Computing**                                                         | $bursaries_postgraduate_computing$  | $scholarships_computing$          |
```

File: app/views/content/funding-and-support/scholarships-and-bursaries.md [hunk 3]
```diff
@@ -43,19 +43,19 @@ You cannot receive a scholarship or bursary if you're on a [salaried teacher tra
...
+| **Design and technology**                                             | $bursaries_postgraduate_designandtechnology$  |                  |
+| **English**                                                           | $bursaries_postgraduate_english$  |                  |
+| **Geography**                                                         | $bursaries_postgraduate_geography$  |                  |
+| **Languages**<br/> (French, German and Spanish only)                  | $bursaries_postgraduate_languagesfrenchgermanspanish$  | £27,000          |
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/scholarships-and-bursaries.md:53]:
```suggestion
| **Languages**<br/> (French, German and Spanish only)                  | $bursaries_postgraduate_languagesfrenchgermanspanish$  | $scholarships_languagesfrenchgermanspanish$      |
```

File: app/views/content/funding-and-support/scholarships-and-bursaries.md [hunk 4]
```diff
@@ -43,19 +43,19 @@ You cannot receive a scholarship or bursary if you're on a [salaried teacher tra
...
+| **Languages**<br/> (all other languages, including ancient languages) | $bursaries_postgraduate_languagesother$  |                  |
+| **Maths**                                                             | $bursaries_postgraduate_maths$  | £30,000          |
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/scholarships-and-bursaries.md:55]:
```suggestion
| **Maths**                                                             | $bursaries_postgraduate_maths$  | $scholarships_maths$         |
```

File: app/views/content/funding-and-support/scholarships-and-bursaries.md [hunk 5]
```diff
@@ -43,19 +43,19 @@ You cannot receive a scholarship or bursary if you're on a [salaried teacher tra
...
+| **Music**                                                             | $bursaries_postgraduate_music$  |                  |
+| **Physics**                                                           | $bursaries_postgraduate_physics$  | £30,000          |
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/scholarships-and-bursaries.md:57]:
```suggestion
| **Physics**                                                           | $bursaries_postgraduate_physics$  | $scholarships_physics$        |
```

**Review by gemmadallmandfe — COMMENTED** [review]
@Sarah-DfE This looks good to me - can you 2i it on Monday so we can merge it?

**Review by Sarah-DfE — APPROVED** [review]
Double-checked and all looks good to me @gemmadallmandfe

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**